### DISCUSSION TITLE
Command line use

### DISCUSSION THREAD

**Discussion by jglazer63** [RandomEngy/VidCoder#948] [General]
I cannot seem to find the documentation on how to use the command line to add items to be encoded.  Can anyone point me to the right place?

**Comment by RandomEngy** [comment]
If you go to `C:\Program Files\VidCoder` and type `.\VidCoderCLI.exe` it spits out the usage:

> Usage:
> VidCoderCLI encode -s[ource] "<source path>" [-d[estination] "<encode file destination>"] -p[reset] "<preset name>" [-picker "<picker name>"]
> VidCoderCLI scan -s[ource] "<source path>"
> VidCoderCLI importpreset "<preset file path>"
> VidCoderCLI importqueue "<queue file path>"

**Reply by jglazer63** [reply]
Thank you!
